ciphers/morse_code.py

```diff
@@ -6,19 +6,61 @@
 https://en.wikipedia.org/wiki/Morse_code
 """
 
-# fmt: off
 MORSE_CODE_DICT = {
-    "A": ".-", "B": "-...", "C": "-.-.", "D": "-..", "E": ".", "F": "..-.", "G": "--.",
-    "H": "....", "I": "..", "J": ".---", "K": "-.-", "L": ".-..", "M": "--", "N": "-.",
-    "O": "---", "P": ".--.", "Q": "--.-", "R": ".-.", "S": "...", "T": "-", "U": "..-",
-    "V": "...-", "W": ".--", "X": "-..-", "Y": "-.--", "Z": "--..", "1": ".----",
-    "2": "..---", "3": "...--", "4": "....-", "5": ".....", "6": "-....", "7": "--...",
-    "8": "---..", "9": "----.", "0": "-----", "&": ".-...", "@": ".--.-.",
-    ":": "---...", ",": "--..--", ".": ".-.-.-", "'": ".----.", '"': ".-..-.",
-    "?": "..--..", "/": "-..-.", "=": "-...-", "+": ".-.-.", "-": "-....-",
-    "(": "-.--.", ")": "-.--.-", "!": "-.-.--", " ": "/"
-}  # Exclamation mark is not in ITU-R recommendation
-# fmt: on
```

Comment:
The current code uses `# fmt: off` and `#fmt: on` properly and I see no reason to mess with that.